### ISSUE TITLE
bug: vendor prefixed CSS is not being built

### ISSUE THREAD

**Issue by EinfachHans** [ionic-team/ionic-framework#22442]
# Bug Report

**Ionic version:**

[ ] **4.x**
[x] **5.x**

**Current behavior:**

After upgrading to 5.4.2 everything is selectable in my iOS App. While i see this in another 5.4.1 app:
```
html.plt-mobile ion-app {
  -webkit-user-select: none;
}
```

and can't find any similar in the 5.4.2 app.

**Expected behavior:**

Content should only be selectable with `contenteditable` like the change in 5.4.2 suggested.
Of course a workaround is easy to do, but i think it should be how it was before?

**Other information:**

**Ionic info:**

```
Ionic:

   Ionic CLI                     : 6.12.1 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework               : @ionic/angular 5.4.2
   @angular-devkit/build-angular : 0.1002.0
   @angular-devkit/schematics    : 10.2.0
   @angular/cli                  : 10.2.0
   @ionic/angular-toolkit        : 2.3.3

Cordova:

   Cordova CLI       : 10.0.0
   Cordova Platforms : android 9.0.0, ios 6.1.1
   Cordova Plugins   : cordova-plugin-ionic-keyboard 2.2.0, cordova-plugin-ionic-webview 5.0.0, (and 27 other plugins)

Utility:

   cordova-res (update available: 0.15.2) : 0.15.1
   native-run (update available: 1.2.2)   : 1.0.0

System:

   Android SDK Tools : 25.2.3 (/Users/hanskrywalsky/Library/Android/sdk)
   ios-deploy        : 1.10.0
   ios-sim           : 8.0.2
   NodeJS            : v12.18.3 (/usr/local/bin/node)
   npm               : 6.14.8
   OS                : macOS Catalina
   Xcode             : Xcode 12.1 Build version 12A7403
```

**Comment by liamdebeasi** [comment]
Can you provide a code reproduction? The contenteditable fix specifically targets only `contenteditable` elements: https://github.com/ionic-team/ionic-framework/commit/023fb1841259a61b361066ca369aeffd488efa3f

**Comment by EinfachHans** [comment]
Repo: https://github.com/EinfachHans/selectable-bug
just try to copy the title

I tested the repo once with `5.4.1` and with `5.4.2` on Emulator (iOS 14.1) and on Device (iOS 14.2).

i mentioned that in the **5.4.1** version the style becomes to:
```
html.plt-mobile ion-app {
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}
```

while on **5.4.2** it is only 
```
html.plt-mobile ion-app {
  user-select: none;
}
```
which is crossed out of course

**Comment by liamdebeasi** [comment]
Thanks, this has been fixed in Ionic Framework v5.4.3.
